### PR TITLE
Add color theme "liqube.theme-liqube-dark".

### PR DESCRIPTION
**Pull request by houhoulis**
Thank you.

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -809,6 +809,12 @@
       "id": "lintangwisesa.arduino",
       "repository": "https://github.com/LintangWisesa/VScode-Arduino-Theme"
     },
+    {
+      "id": "liqube.theme-liqube-dark",
+      "repository": "https://github.com/liqube/liqube-dark-vscode",
+      "version": "1.0.5",
+      "checkout": "master"
```
Comment (jankeromnes) [inline extensions.json:816]: Unfortunately, this repository doesn't provide any GitHub Releases or release tags. So, the best remaining option is to publish straight from the default branch (no need to explicitly check it out, and no need to pin a specific version):

```suggestion
      "repository": "https://github.com/liqube/liqube-dark-vscode"
```

**Review by jankeromnes — APPROVED** [review]
Approved, thanks!

Minor edit suggestion in-line, but I can apply it myself on GitHub before merging, so this is good to go. 🚀

**Comment by houhoulis** [comment]
Thanks for the fix; sorry I missed it. 👍🏼

**Comment by houhoulis** [comment]
I should mention, FYI, I generated this patch by signing up for Gitpod and using your Gitpod button tool. I'm not sure if that indicates a problem with the tool.

**Comment by jankeromnes** [comment]
Sure! And no worries. Your usage of the tool was perfect, it's just a false positive (a rare case where the repository does not provide `.vsix` in GitHub Releases or suitable release tags -- but you're right, we could/should probably handle that corner case in the `add-extension` script).

**Comment by jankeromnes** [comment]
Thanks again @houhoulis!

The extension was auto-published successfully: https://open-vsx.org/extension/liqube/theme-liqube-dark 🎉

Could you please confirm that it works as expected?

**Comment by houhoulis** [comment]
It does! Much appreciated.

**Pull request merged**